docs/sources/mimir/get-started/_index.md

```diff
@@ -227,5 +227,4 @@ To add a data source, refer to [Add a data source](/docs/grafana/latest/administ
 
 ## Verify success
 
-After you have completed the tasks in this _Get started_ guide, you can query metrics in [Grafana Explore](/docs/grafana/latest/explore/)
-as well as create dashboard panels using your newly configured Grafana Mimir data source.
+After you have completed the tasks in this _Get started_ guide, you can query metrics in [Grafana Explore](/docs/grafana/latest/explore/) and create dashboard panels using your newly configured Grafana Mimir data source.
```

Comment:
Is this change intentional?

Reply:
Sorry, it's a trivial change to trigger the deploy preview (to prove that it works). I made it in a separate [commit](https://github.com/grafana/mimir/pull/11338/commits/8723d0ae38fa86cfc327cc2c300af5272451057f) that I can drop if you'd prefer